src/components/AllMovies/AllMovies.js

```diff
@@ -27,4 +29,12 @@ function AllMovies( {movies , onMovieClick} ){
 	)
 }
 
-export default AllMovies;
\ No newline at end of file
+export default AllMovies;
+
+AllMovies.propTypes = {
+  movies: PropTypes.array.isRequired,
```

Comment:
PropTypes on complex data types (arrays, objects) need to be deeply validated. What is the data types of the elements inside the array/object?